### DISCUSSION TITLE
element plus版本路线图是啥样子的？

### DISCUSSION THREAD

**Discussion by kimmy-wang** [element-plus/element-plus#8927] [Q&A]
@jw-foss element plus版本路线图是啥样子的？element plus 3.0啥时候发布呢？据说3.0要重构仓库为mono repo项目

**Comment by bbcvc** [comment]
看下这个discussions，有讲了一下element-plus下一步打算
https://github.com/element-plus/element-plus/discussions/8288

**Reply by kimmy-wang** [reply]
我的意思是3.x版本啥时候发布？
